.gitignore

```diff
@@ -4,3 +4,4 @@ dw.json
 .DS_Store/
 cartridges/app_storefront_base/
 cartridges/modules/
+cartridges/app_custom2/cartridge/static/
```

Comment:
change to look for only *.css and *.js files, no need to exclude the whole static folder as it might include images, which need to be pushed